src/main/java/org/zalando/jackson/datatype/money/DefaultLocaleProviderImpl.java

```diff
@@ -0,0 +1,16 @@
+package org.zalando.jackson.datatype.money;
+
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.util.Locale;
+
+/**
+ * @author Petar Tahchiev
+ * @since 1.5
+ */
+public class DefaultLocaleProviderImpl implements LocaleProvider {
```

Comment:
Can you drop the `Impl` suffix? Final would also be nice. We can always open it up later, if needed.

Reply:
Package private would also be better, for now.